src/model/services/mapi/rbacv1/types.ts

```diff
@@ -84,7 +84,7 @@ export interface IRoleBinding {
   kind: typeof RoleBinding;
   metadata: metav1.IObjectMeta;
   roleRef: IRoleRef;
-  subjects: ISubject[];
+  subjects?: ISubject[];
```

Comment:
From what I understand [here](https://github.com/kubernetes/api/blob/b8c40e080bc5e830097df540d4ef804034cb5bdb/rbac/v1/types.go#L131), `subjects` can be omitted if empty